src/cplscheme/config/CouplingSchemeConfiguration.cpp

```diff
@@ -984,23 +980,32 @@ void CouplingSchemeConfiguration::addDataToBeExchanged(
     if (from == accessor) {
       scheme.addDataToSend(exchange.data, exchange.mesh, requiresInitialization);
       if (requiresInitialization && (_config.type == VALUE_SERIAL_EXPLICIT || _config.type == VALUE_SERIAL_IMPLICIT)) {
-        PRECICE_CHECK(not scheme.doesFirstStep(),
-                      "In serial coupling only second participant can initialize data and send it. "
-                      "Please check the <exchange data=\"{}\" mesh=\"{}\" from=\"{}\" to=\"{}\" initialize=\"{}\" /> tag in the <coupling-scheme:... /> of your precice-config.xml.",
-                      dataName, meshName, from, to, requiresInitialization);
+        if (_experimental && scheme.doesFirstStep()) {
+          PRECICE_WARN("In serial coupling first participant can only initialize data, if the experimental interface is active. This is only useful for time interpolation of order 1 or higher. Please consider checking the documentation for more details.");
```

Comment:
I don't understand the first sentence. Could you rewrite?

Reply:
I think we can just remove this warning. If the experimental interface is not activated the user gets an error anyway (we might also soon move the waveform features from the experimental interface to the real interface anyway). Removed it in c985cb315162b305a73efbe7575ba334d7dde281